global.json

```diff
@@ -1,12 +1,12 @@
 {
   "tools": {
-    "dotnet": "2.1.401",
+    "dotnet": "2.1.602-preview-009587",
     "vs": {
       "version": "15.8"
```

Comment:
I expected this to also change, but moving it to 16.0 broke. Is there a more-correct way to say "this repo only builds in VS 16.0+"?

Reply:
If you specify 
```
"vs": {
  "version": "16.0"
}
```

It will require VS 16.0+. 

If the CI machine does not have it installed you can also add

```
"xcopy-msbuild": "16.0.0-rc1-alpha"
```

to fall back to xcopy msbuild package.

Reply:
That said, the Arcade changes shouldn't be the reason to require 16.

Reply:
`"version": "16.0"` fails, for example: https://dev.azure.com/dnceng/public/_build/results?buildId=128638

```
dotnet-install: Installation finished
Downloading vswhere
Downloading RoslynTools.MSBuild 16.0.0-alpha
No package matching package identifier RoslynTools.MSBuild and version 16.0.0-alpha was found in the given feed.
##[error]Process completed with exit code 1.
```

Maybe that's worked around by the xcopy-msbuild specification? I can try that.

Reply:
> That said, the Arcade changes shouldn't be the reason to require 16.

Sidestepping this for now by moving to a 2.1.5xx SDK.

Reply:
> Maybe that's worked around by the xcopy-msbuild specification?

Yes. The script infers default name for the package from the specified VS version "16.0", but the actual package name also includes "rc1" which the script does not know it should include. Specifying the package name explicitly would fix that.